plasmapy/formulary/mathematics.py

```diff
@@ -92,3 +92,43 @@ def Fermi_integral(
         return integral_arr
     else:
         raise TypeError(f"Improper type {type(x)} given for argument x.")
+
+
+def rot_a_to_b(a, b):
+    r"""
+    Calculates the 3D rotation matrix that will rotate vector a to be aligned
+    with vector b.
+
+    Parameters
+    ----------
+    a : ~np.ndarray, shape (3)
+        Pre-rotation orientation unit vector
+
+    b : ~np.ndarray, shape (3)
+        Post-rotation orientation unit vector
```

Comment:
```suggestion
    b : `~numpy.ndarray`, shape (3,)
        Vector representing the desired orientation after rotation.  Should be
        a 1D, 3-element unit vector.  If ``b`` is not normalized, then it will
        be.
```